### ISSUE TITLE
Adaptation to Wan2.2-TI2V

### ISSUE THREAD

**Issue by Lewis-lv0** [shengshu-ai/minWM#14]
Thank you for open-sourcing this great project! 

I am integrating the Wan2.2-TI2V backbone, so I am mostly follow the training dynamics of Wan2.1 in this project. May I know the number of training steps that is required for Stage0, 1, 2, 3 to converge? Seems that this is not shown in the code right now.

I am also curious if I can scale the learning rate accordingly if I have a larger global batch size, say 64.

**Comment by zhuhz22** [comment]
You can refer to our technical report for the training steps, but note that Wan2.2 (14B/5B) is larger than Wan 2.1 (1.3B), so it may take more steps to converge.

For `lr`, you can try to scalre since the gbs is large.
